docs/production-deployment/cloud/api-keys.md

```diff
@@ -78,6 +79,25 @@ For a complete example, see [Cloud Samples in Go](https://github.com/temporalio/
 
 Manage your personal API Keys by creating, deleting, or updating access to them using the Cloud UI or tcld.
 
+### Global Administrator API Key Management
+
+:::note
+
+API Keys are not enabled for by default.
+You must enable API Key access to allow the creation of API Keys.
+
+:::
+
+Global Administrators can monitor, manage, update access, and delete API Keys for any user within their account.
+
+To manage your accounts API Keys
+
+1. [Login](https://cloud.temporal.io/) to the Cloud UI.
+2. Select **Settings** and choose **API Keys**.
```

Comment:
pls add a link to this line so users can click and go to Settings-> API Keys 
https://cloud.temporal.io/settings/api-keys